pkg/registry/scripts/projects.js

```diff
@@ -128,7 +167,13 @@
                 var display = fields.display.trim();
                 var description = fields.description.trim();
 
-                if (!NAME_RE.test(name)) {
+                if (!name) {
```

Comment:
Doesn't the NAME_RE already handle the case of empty names? If not, lets just adjust NAME_RE to not match empty strings.